From 66a794b0eeda8c293e29614c2072f265586d8805 Mon Sep 17 00:00:00 2001
From: Ben Cressey <bcressey@amazon.com>
Date: Sat, 10 Aug 2019 18:31:42 +0000
Subject: [PATCH 3/4] ship mkconst and schema sources for runtime use

`mkconst` needs to be compiled for the host architecture, but relies
on most of the sources in wicked and needs all of the same libraries.

This is a much greater dependency on the host OS than we care to take.
For now we punt the problem to runtime and expect `constants.xml` to
be created out-of-band.

Signed-off-by: Ben Cressey <bcressey@amazon.com>
---
 schema/Makefile.am | 2 +-
 util/Makefile.am   | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/schema/Makefile.am b/schema/Makefile.am
index 462dc044..03a4bcd3 100644
--- a/schema/Makefile.am
+++ b/schema/Makefile.am
@@ -4,7 +4,7 @@ MAINTAINERCLEANFILES		= Makefile.in
 
 wicked_schema_DATA		= \
 	$(wicked_schema_files)	\
-	$(wicked_schema_built)
+	$(wicked_schema_sources)
 
 wicked_schema_files		= \
 	addrconf.xml		\
diff --git a/util/Makefile.am b/util/Makefile.am
index 2af1435b..bb9697e1 100644
--- a/util/Makefile.am
+++ b/util/Makefile.am
@@ -2,7 +2,8 @@
 CLEANFILES			= *~
 MAINTAINERCLEANFILES		= Makefile.in
 
-noinst_PROGRAMS			= mkconst schema2html
+wicked_sbin_PROGRAMS		= mkconst
+noinst_PROGRAMS			= schema2html
 
 AM_CPPFLAGS			= -I$(top_srcdir)/src	\
 				  -I$(top_srcdir)/include
-- 
2.32.0

